Snakefile

```diff
@@ -34,8 +34,8 @@ files = rules.files.params
 rule download:
     message: "Downloading metadata and fasta files from S3"
     output:
-        sequences = "data/sequences.fasta",
-        metadata = "data/metadata.tsv"
+        sequences = config["sequences"],
+        metadata = config["metadata"]
     shell:
         """
         aws s3 cp s3://nextstrain-ncov-private/sequences.fasta data/
```

Comment:
The shell commands run here also need to be updated to use `{output.sequences:q}` and `{output.metadata:q}`.